### ISSUE TITLE
Links to social media posts

### ISSUE THREAD

**Issue by pixeljitsu** [engram-design/SocialPoster#3]
It would be really handy if there were a way to access the links leading to the social media posts from within the entry.  I'd like to automatically display the links as part of the craft entry.  "Join the discussion on Facebook/Twitter", etc.

**Comment by engram-design** [comment]
Just to confirm, this would be from the front-end? This way, you could post the entry to Facebook, have it record the posted URL, and then being able to access in your template?

Something like:

```
{% set post = craft.socialPoster.post({ account: 'facebook', entryId: entry.id }) %}

{% if post.url %}
    <a href="{{ post.url }}" target="_blank">Join the discussion on Facebook</a>
{% endif %}
```

**Comment by pixeljitsu** [comment]
Exactly.  That way you could drive traffic from the website to the social media posts.

It'd also be nice to loop through all of the accounts. Something like this:

```
{% set posts = craft.socialPoster.post({ entryId: entry.id }) %}

{% for post in posts %}
    <a href="{{ post.url }}" target="_blank">
        <i class="fa fa-{{ post.type }}-square"></i> Join the discussion on {{ post.type | capitalize }}
    </a>
{% endfor %}
```

or something close to that.

**Comment by engram-design** [comment]
Just to follow this up - this is now implemented in [1.2.0](https://github.com/engram-design/SocialPoster/releases/tag/1.2.0). Use the following template code:

```
{% set posts = craft.socialPoster.posts({ element: entry }) %}

{% for post in posts %}
    <a href="{{ post.url }}" target="_blank">
        <i class="fa fa-{{ post.handle }}-square"></i> Join the discussion on {{ post.handle | capitalize }}
    </a>
{% endfor %}
```
